model/src/main/java/com/expediagroup/streamplatform/streamregistry/model/Zone.java

```diff
@@ -30,4 +30,9 @@
   private ZoneKey key;
   private Specification specification;
   private Status status;
+
```

Comment:
Looks like we're missing a `ZoneResolver`. Perhaps create one so that the solution for the status is consistent across all entities?

Reply:
done